templates/client.json.j2

```diff
@@ -1,7 +1,7 @@
 {
   "client": {
-    "name": "{{ ansible_hostname }}",
+    "name": "{{ sensu_client_name }}",
     "address": "{{ ansible_default_ipv4['address'] }}",
-      "subscriptions": {{ group_names | to_nice_json }}
+    "subscriptions": {{ sensu_client_subscriptions | to_nice_json(indent=6) }}
```

Comment:
It was incorrect before. It's not strictly necessary, but I ended up fixing while testing it out. Feel free to revert it if you prefer :slightly_smiling_face: